README.md

```diff
@@ -1,4 +1,4 @@
-# Plugin Starter Template ![CircleCI branch](https://img.shields.io/circleci/project/github/mattermost/mattermost-plugin-starter-template/master.svg)
+# Plugin Starter Template [![](https://circleci.com/gh/mattermost/mattermost-plugin-starter-template.svg?style=shield)](https://circleci.com/gh/mattermost/mattermost-plugin-starter-template)
```

Comment:
I like the addition of the link to circleci.

What is your reasoning for not fetching the icon from shields.io? I find it quite handy to get all icons from the same source in the same style.

Reply:
While I am really sure the design can be unified with shield.io, I do not prefer shield.io badges due to its stability. e.g., badges/shields#1518 for Code Climate case.

Also ![](https://circleci.com/gh/mattermost/mattermost-plugin-starter-template.svg?style=shield) can be acceptable to me.

Reply:
Then let's just stick with the icon from shield.io. I like the unified source. Would you be fine with this?

Reply:
@hanzei Thanks. Do you mean you like the following?

```suggestion
# Plugin Starter Template [![](https://img.shields.io/circleci/project/github/mattermost/mattermost-plugin-starter-template/master.svg)](https://circleci.com/gh/mattermost/mattermost-plugin-starter-template)
```